### ISSUE TITLE
NVDA does not identify action notifications by control surface

### ISSUE THREAD

**Issue by luanmusical** [jcsteh/osara#746]
Translation:
Hello! I've been using the beringer x-touch compact control surface here together with reaper through the DrivenByMoss4Reaper-15.6.0-Windows drive and it has been working fine, it wasn't a problem for example when I press the send button on the surface it displays a message temp on screen where nvda reads only notification without reading message content! as it is a small temporary window that disappears quickly, I cannot use different navigation modes to try to access this content. it would be interesting for this content to be read automatically as nvda displays the notification message as this would be quite productive.
for information these messages are for example the name and number of the send, if the surface is in volume or panorama mode, to which send are you currently directed!!!

Original (preserved in case a non-automated translation is needed):
olá! tenho usado por aqui juntamente com o reaper a superficie de controle beringer x-toutch compact através do drive DrivenByMoss4Reaper-15.6.0-Windows e esta tem funcionado bem, não fosse um problema por exemplo quando pressiono o botão de send na superficie exibe uma mensagem temporária na tela onde o nvda lê apenas notification sem ler o conteúdo da mensagem! como se trata de uma pequena janela temporária que desaparece rapidamente não consigo me utilizar de modos de navegação diferentes para tentar acessar esse conteúdo. seria interessante  que esse conteúdo fosse lído automaticamente assim como o nvda exibe a mensagem notification  pois isso seria bastante producente.
para informação essas mensagens são por exemplo o nome e numero do send, se a superficie está no modo de volumes ou panorama, pra qual send você está direcionado no momento!!!

**Comment by ScottChesworth** [comment]
Please translate into English before posting. Automated translation is fine if you don't speak English. Thanks.

**Comment by ScottChesworth** [comment]
Haven't added a priority label yet because I've got no idea what this type of work might look like. Have any of our contributors used DrivenByMoss?

**Comment by luanmusical** [comment]
Sorry for the post in Portuguese!

**Comment by jcsteh** [comment]
This is almost certainly a custom window displayed by DrivenByMoss. OSARA can't make this accessible.

DrivenByMoss could use the osara_outputMessage API to report the message to OSARA, but this change would have to be made in DrivenByMoss.

**Comment by jcsteh** [comment]
Alternatively, an NVDA add-on could be written to make this accessible, but that's out of scope for the OSARA project.
